### ISSUE TITLE
API Extractor cannot find name OffscreenCanvas

### ISSUE THREAD

**Issue by jasonsturges** [pixijs/pixijs#8174]
Hi, I'm trying to publish a TypeScript library npm package that includes Pixi, and Microsoft API Extractor is failing `OffscreenCanvas`

```
build:types
> tsc -p ./tsconfig.json --outDir build --declaration true && api-extractor run

node_modules/@pixi/core/index.d.ts:1543:45 - error TS2304: Cannot find name 'OffscreenCanvas'.

1543     static test(source: unknown): source is OffscreenCanvas | HTMLCanvasElement;
                                                 ~~~~~~~~~~~~~~~

node_modules/@pixi/text/index.d.ts:272:41 - error TS2304: Cannot find name 'OffscreenCanvas'.

272     static _canvas: HTMLCanvasElement | OffscreenCanvas;
                                            ~~~~~~~~~~~~~~~

node_modules/@pixi/text/index.d.ts:273:49 - error TS2304: Cannot find name 'OffscreenCanvasRenderingContext2D'.

273     static _context: CanvasRenderingContext2D | OffscreenCanvasRenderingContext2D;
                                                    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

node_modules/@pixi/text/index.d.ts:300:105 - error TS2304: Cannot find name 'OffscreenCanvas'.

300     static measureText(text: string, style: TextStyle, wordWrap?: boolean, canvas?: HTMLCanvasElement | OffscreenCanvas): TextMetrics_2;
                                                                                                            ~~~~~~~~~~~~~~~
```

Think there's some movement with offscreen canvas, maybe this has already been addressed.  

FYI, or please advise if this seems in error.

### Environment

- **pixi.js**:  6.2.2
- **tslib**: 2.3.1
- **TypeScript**: 4.5.5
- **@microsoft/api-extractor**: 7.19.4
- **OS & Version**: macOS Monterey, Apple M1 Pro

**Comment by bigtimebuddy** [comment]
I've stubbed my toe on this too. If you add @types/OffscreenCanvas to your project it should fix. But we should add this as a dependency.

**Comment by jasonsturges** [comment]
@bigtimebuddy Awesome, thank you!

**Comment by AlanSean** [comment]
> 我也在这件事上扎了脚。如果您将 @types/OffscreenCanvas 添加到您的项目中，它应该可以修复。但是我们应该将其添加为依赖项。
It should be@types/offscreenCanvas

**Comment by kevincentius** [comment]
I am getting this error even after adding @types/offscreencanvas?

I am using pixi js within an angular application.

**Comment by kevincentius** [comment]
Hello, I am getting this exact error with pixi 6.3.2.
Adding `@types/offscreencanvas` did not help.
Someone else seems to have it too: https://stackoverflow.com/a/71695828/2015374

**Environment**
pixi.js: 6.3.2
tslib: 2.3.0
TypeScript: 4.6.2
Windows 10

I am using pixi in an angular project, I don't know if it makes any difference.
I don't know what @microsoft/api-extractor is, I tried installing it (7.23.1) but nothing changed.

For now I switched to pixi 5.3.12 to avoid this error.

**Comment by kevincentius** [comment]
Hello, I am getting this exact error with pixi 6.3.2.
Installing `@types/offscreencanvas` did not help.
Looks like someone else has the same problem: https://stackoverflow.com/a/71695828/2015374

Environment
pixi.js: 6.3.2
tslib: 2.3.0
TypeScript: 4.6.2
Windows 10

I am using pixi in an angular project, I don't know if it makes any difference.
I don't know what @microsoft/api-extractor is, I tried installing it (7.23.1) but nothing changed.

For now I switched to pixi 5.3.12 to avoid this error.
